protocol/osrs-225/osrs-225-api/src/main/kotlin/net/rsprot/protocol/api/traffic/GenericNetworkTrafficWriter.kt

```diff
@@ -0,0 +1,218 @@
+package net.rsprot.protocol.api.traffic
+
+import net.rsprot.protocol.loginprot.incoming.util.HostPlatformStats
+import net.rsprot.protocol.loginprot.incoming.util.LoginBlock
+import net.rsprot.protocol.metrics.channel.snapshots.impl.GenericChannelTrafficSnapshot
+import net.rsprot.protocol.metrics.channel.snapshots.util.PacketSnapshot
+import net.rsprot.protocol.metrics.snapshots.impl.GenericNetworkTrafficSnapshot
+import net.rsprot.protocol.metrics.writer.NetworkTrafficWriter
+import java.net.InetAddress
+import java.text.NumberFormat
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+import java.time.format.FormatStyle
+import kotlin.collections.iterator
+
+public data object GenericNetworkTrafficWriter : NetworkTrafficWriter<GenericNetworkTrafficSnapshot<*>, String> {
+    private const val INDENT: String = "  "
```

Comment:
y u no `"\t"`

Reply:
.... maybe that's a controversial topic 😂

Reply:
Tabs bad. Two spaces great. End of conversation!